BlueprintUI/Sources/Layout/Builder.swift

```diff
@@ -44,4 +44,23 @@ public struct Builder<Child> {
     public static func buildExpression(_ children: [Child]) -> Children {
         children
     }
+
+    /// Allow an escape hatch when the control flow requires an output (such as a `switch`).
+    public static func buildExpression(_ children: None) -> Children {
+        []
+    }
+}
+
+/// Empty type that results in no children. This is useful when an expression requires output but no children are applicable (i.e. a switch statement).
+/// Example:
+/// ```
+/// switch item {
+/// case .first:
+///    None()
+/// case .second:
+///     "2"
+/// }
+/// ```
+public struct None {
```

Comment:
Curious what other folks on the team think about this name; could also see `BuildNone` or `BuildNothing` being nice, since it hints at the result builder use-case and is perhaps more clear when reading?

Reply:
I'm open to that! Definitely not sold on the name since it is a bit vague.

Reply:
I was also thinking `Nothing` could work

Reply:
I'm good with `Nothing`. I wonder if the prefix `Build` would be a little less intuitive because most other declarations inside the builder do not use a verb as a prefix, they just declare the noun. Though `BuildNothing` would be less likely to conflict with other libraries?

Reply:
No strong feeling here. It feels rare enough that it's not super critical either way

Reply:
Changed to `Nothing`